compiler/src/iree/compiler/Codegen/SPIRV/VideoCoreConfig.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright 2024 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+//===- VideoCoreConfig.h - VideoCore CodeGen Configurations ---------------------===//
+//
+// This file contains CodeGen configurations for Broadcom VideoCore GPUs.
+//
+//===----------------------------------------------------------------------===//
+
+#include <array>
+
+#include "iree/compiler/Codegen/SPIRV/KernelConfig.h"
+#include "mlir/Dialect/Linalg/IR/Linalg.h"
+
+namespace mlir::iree_compiler::detail {
+
+static LogicalResult setVideoCoreMatmulConfig(linalg::LinalgOp op,
+                                           IREE::GPU::TargetAttr target) {
+  const std::array<int64_t, 2> workgroupXY = {16, 16};
```

Comment:
Can you add a check for f32 here? We didn't investigate f16 yet and should fall back